sycl/doc/GetStartedGuide.md

```diff
@@ -663,6 +699,12 @@ which contains all the symbols required.
 * The NVIDIA OpenCL headers conflict with the OpenCL headers required for this
   project and may cause compilation issues on some platforms
 
+### ROCm back-end limitations
+
+* Backend is only supported on Linux
```

Comment:
Would it be fair to say, that ROCm is only supported on Linux? I.e. that is the blocker for ROCm backend on other platforms.

Reply:
I have only tried it on Linux, and not sure whether it is normal on other platforms.